app/views/overview/_service-group.html

```diff
@@ -50,14 +50,16 @@ <h4 class="route-title truncate">
           <!--   <span class="pficon pficon pficon-add-circle-o" aria-hidden="true"></span> -->
           <!-- </div> -->
           <h2>No linked services.</h2>
-          <p>
-            No services are linked to {{service.metadata.name}}.
-            <span ng-if="(services | hashSize) > 1">Add a service to group them together.</span>
-          </p>
-          <div ng-if="(services | hashSize) > 1">
-            <button class="btn btn-primary" ng-show="(services | hashSize) > 1" ng-click="linkService()">
-              Link Service
-            </button>
+          <div ng-if="'services' | canI : 'update'">
```

Comment:
the latter feels more balanced even though a little redundant, lets go with
the second one.  It would be better if "frontend" was a link to the service
to be consistent with the no deployments message, but we can fix that
outside of this PR  @spadgett (later so it doesn't conflict with these
changes)

On Mon, Jul 11, 2016 at 5:45 AM, Jakub Hadvig <notifications@github.com>
wrote:

> In app/views/overview/_service-group.html
> <https://github.com/openshift/origin-web-console/pull/82#discussion_r70228757>
> :
>
> > @@ -50,14 +50,16 @@ <h4 class="route-title truncate">
> >            <!--   <span class="pficon pficon pficon-add-circle-o" aria-hidden="true"></span> -->
> >            <!-- </div> -->
> >            <h2>No linked services.</h2>
> > -          <p>
> > -            No services are linked to {{service.metadata.name}}.
> > -            <span ng-if="(services | hashSize) > 1">Add a service to group them together.</span>
> > -          </p>
> > -          <div ng-if="(services | hashSize) > 1">
> > -            <button class="btn btn-primary" ng-show="(services | hashSize) > 1" ng-click="linkService()">
> > -              Link Service
> > -            </button>
> > +          <div ng-if="'services' | canI : 'update'">
>
> @jwforres <https://github.com/jwforres>
> [image: screenshot]
> <https://cloud.githubusercontent.com/assets/1668218/16726490/c08d3da2-475c-11e6-931d-a6117c734d2d.png>
> vs.
> [image: screenshot-2]
> <https://cloud.githubusercontent.com/assets/1668218/16726496/c6f39f38-475c-11e6-90c1-f22673457c44.png>
>
> Thinking about it now I would leave the "No services are linked to foo."
> there. Thoughts ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin-web-console/pull/82/files/a7e24db8d96bfa2c18c40ed662c2b8a3cdf38a80#r70228757>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABZk7fs5yshWhcpQ_IC_Q42VDJw_b1T_ks5qUhCugaJpZM4I6BvO>
> .
>